allauth/socialaccount/providers/gitlab/provider.py

```diff
@@ -12,7 +12,7 @@ def get_avatar_url(self):
 
     def to_str(self):
         dflt = super(GitLabAccount, self).to_str()
-        return self.account.extra_data.get("name", dflt)
+        return self.account.extra_data.get("username", dflt)
```

Comment:
`username` is already part of the base class, so I would expect this to be superfluous?